AlfrescoSDK/AlfrescoSDK/Constants/AlfrescoConstants.h

```diff
@@ -314,3 +314,20 @@ extern NSString * const kAlfrescoWorkflowVariableTaskInvitePendingOutcome;
 extern NSString * const kAlfrescoWorkflowTaskTransitionApprove;
 extern NSString * const kAlfrescoWorkflowTaskTransitionAccept;
 extern NSString * const kAlfrescoWorkflowTaskTransitionReject;
+
+/**---------------------------------------------------------------------------------------
+ * @name Connection Diagnostic Constants
+ --------------------------------------------------------------------------------------- */
+extern NSString * const kAlfrescoConfigurationDiagnosticDidStartEventNotification;
+extern NSString * const kAlfrescoConfigurationDiagnosticDidEndEventNotification;
+
+extern NSString * const kAlfrescoConfigurationDiagnosticDictionaryIsLoading;
+extern NSString * const kAlfrescoConfigurationDiagnosticDictionaryIsSuccess;
```

Comment:
I was thinking something along the lines of:
"status" = "loading"
"status" = "successful"
"status" = "failed"

The values could also be strings.